ui/src/Pages/PeoplePage/PeoplePage.tsx

```
@@ -269,31 +276,61 @@ export const PeoplePage = () => {
+  modals = (
+    <>
+      <MergeFaceGroupsModal
+        state={mergeModalState}
+        setState={setMergeModalState}
+        refetchQueries={[
+          {
+            query: MY_FACES_QUERY,
+          },
+        ]}
+      />
+    </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)